awscli/examples/application-signals/start-discovery.rst

```diff
@@ -0,0 +1,11 @@
+**To enable this Amazon Web Services account to be able to use CloudWatch Application Signals**
+
+The following ``start-discovery`` example enables this Amazon Web Services account to be able to use CloudWatch Application Signals by creating the AWSServiceRoleForCloudWatchApplicationSignals service-linked role. ::
+
+    aws application-signals start-discovery
+
+Output::
```

Comment:
```suggestion
```

Reply:
Can you remove the `Output::` line from the commands that don't have any. It's actually invalid .rst, hence the test failures.